core/security/src/main/java/org/apache/isis/core/security/authorization/manager/AuthorizationManager.java

```diff
@@ -50,7 +50,6 @@ public class AuthorizationManager {
     @Inject
     public AuthorizationManager(
             final List<Authorizor> authorizors,
-            // TODO: elsewhere we inject an Optional<X>, should use the same technique throughout...
             @org.springframework.lang.Nullable final AuthorizorChooser authorizorChooser) {
```

Comment:
TODO: actually use Optional<X> here ... as this is the more common idiom within the framework, which does not depend on specific Spring annotations (that are less robust regarding accidental refactoring sloppiness)